### PR TITLE
DM-29526: update the code for newer versions of dependencies

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -39,6 +39,10 @@ where = src
 console_scripts =
     exposurelog = exposurelog.cli:main
 
+[tool:pytest]
+filterwarnings =
+    ignore:numpy.ndarray size changed:RuntimeWarning
```
Comment (SimonKrughoff) [inline setup.cfg:44]: Will you add a note to say why this warning is being ignored?

File: src/exposurelog/resolvers/delete_messages.py [hunk 2]
```diff
@@ -17,8 +17,8 @@
 async def delete_messages(
     app: aiohttp.web.Application,
     _info: graphql.GraphQLResolveInfo,
-    **kwargs,
-) -> dict:
+    **kwargs: typing.Any,
+) -> typing.List[dict]:
```
Comment (SimonKrughoff) [inline src/exposurelog/resolvers/delete_messages.py:21]: Now that we are on Python 3.8, I thought you could just use `list[dict]`

File: src/exposurelog/resolvers/find_messages.py [hunk 3]
```diff
@@ -16,8 +16,8 @@
 async def find_messages(
     app: aiohttp.web.Application,
     _info: graphql.GraphQLResolveInfo,
-    **kwargs,
-) -> dict:
+    **kwargs: typing.Any,
+) -> typing.List[dict]:
```
Comment (SimonKrughoff) [inline src/exposurelog/resolvers/find_messages.py:20]: Again, I thought you could just use `list[dict]`

**Review by SimonKrughoff — APPROVED** [review]
Seems good to me.  A couple of small comments.

File: src/exposurelog/resolvers/delete_messages.py (hunk 2 above)
Comment (r-owen) [reply src/exposurelog/resolvers/delete_messages.py:21]: Thanks. I'll make that change. Pity that `dict` doesn't work the same way (I tried).

File: src/exposurelog/resolvers/find_messages.py (hunk 3 above)
Comment (r-owen) [reply src/exposurelog/resolvers/find_messages.py]: I have changed it everywhere.

File: setup.cfg (hunk 1 above)
Comment (r-owen) [reply setup.cfg:44]: Yes. Good suggestion.

**Pull request merged**